src/main/java/com/quizapp/enterprise/persistence/GameTracker.java

```diff
@@ -73,7 +73,12 @@ public Question getNextQuestion(String gameCode){
 
     public void updatePlayerRound(String gameCode, String playerName, boolean correct, boolean setComplete){
         Game game = getGameByCode(gameCode);
-        Player playerToUpdate =  game.getPlayers().stream().filter(player -> player.getPlayerUsername().equals(playerName)).findFirst().get();
+        Player playerToUpdate = game.getPlayers()
+                .stream()
+                .filter(player -> player.getPlayerUsername()
+                        .equals(playerName))
+                .findFirst()
+                .get();
```

Comment:
reformatted the streams to use indentation so the code may be cleaner and easier to read.
This recommended by the "Iterators and Streams" section of the java best practices summary referenced below.

source: https://rhamedy.medium.com/a-short-summary-of-java-coding-best-practices-31283d0167d3